bridge_adaptivity/module/mixins/views.py

```diff
@@ -51,36 +53,73 @@ class GroupEditFormMixin(object):
     prefix = 'group'
     grading_prefix = 'grading'
 
-    def get_grading_form_kwargs(self):
-        """Return kwargs for GradingForm."""
-        form_kw = dict(
-            prefix=self.grading_prefix,
-        )
-        if self.object and self.object.grading_policy:
-            form_kw['instance'] = self.object.grading_policy
+    def get_form(self):
+        form = super().get_form()
+        form.fields['course'].queryset = Course.objects.filter(owner=self.request.user)
+        return form
+
+
+class BaseEditFormMixin(object):
```

Comment:
In Python 3 we don’t need to be inherited from object.

Reply:
You are right. I fixed it.